test/e2e/e2e_suite_test.go

```diff
@@ -42,4 +53,76 @@ var _ = BeforeSuite(func() {
 	Expect(err).ToNot(HaveOccurred())
 	c, err = client.New(cfg, client.Options{Scheme: scheme})
 	Expect(err).To(Not(HaveOccurred()))
+
+	ctx := context.Background()
+	operatorCatalog, err = createTestCatalog(ctx, testCatalogName, testCatalogRef)
+	Expect(err).ToNot(HaveOccurred())
+
+	Eventually(func(g Gomega) {
+		err := c.Get(ctx, types.NamespacedName{Name: operatorCatalog.Name}, operatorCatalog)
+		g.Expect(err).ToNot(HaveOccurred())
+		cond := apimeta.FindStatusCondition(operatorCatalog.Status.Conditions, catalogd.TypeUnpacked)
+		g.Expect(cond).ToNot(BeNil())
+		g.Expect(cond.Status).To(Equal(metav1.ConditionTrue))
+		g.Expect(cond.Reason).To(Equal(catalogd.ReasonUnpackSuccessful))
+
+		// Ensure some packages exist before continuing so the
+		// operators don't get stuck in a bad state
+		pList := &catalogd.PackageList{}
+		err = c.List(ctx, pList)
+		g.Expect(err).ToNot(HaveOccurred())
```

Comment:
For anything that just returns an error, I find this is shorter/easier to parse:
```suggestion
		g.Expect(c.List(ctx, pList)).To(Succeed())
```

Reply:
https://github.com/operator-framework/operator-controller/pull/242/commits/82499689cb2eac6191e184ddd8875be0972e4690

Reply:
Oops, missed some: https://github.com/operator-framework/operator-controller/pull/242/commits/7f4c1bff9d5e053576eddd29d57c063bd0c02b6f